### PR TITLE
Update social approach mechanisms

### PR DESCRIPTION
**Pull request by mackgalante**
Updated mechanisms text with reference to Carey et. al. (2019)

### CODE REVIEW

**Comment by mackgalante** [comment]
Let me comment in what needs review:

File: vignettes/social_approach_vignette/readme.md [hunk 1]
```diff
@@ -95,11 +95,11 @@ Moderators are factors that inhibit or promote the influence of the feedback int
 4. `Slope of trend` - Derivative of the performance measure in the time domain
 
 ### Mechanisms
-Mechanisms are factors that the intervention operates through to influence the feedback recipient. The social approach pathway has the following mechanisms:
-1. `Awareness (knowledge)` - The message may change the recipient's awareness of their performance increasing and nearing a high performance benchmark relative to their peers.
-2. `Subjective norms` - The message may influence the recipient by creating or reinforcing their proximity to top/high-performer status within their peer group.
-3. `Motivation` - The message may motivate the recipient to keep up their recent performance increase to reach new social status.
-4. `Optimism` - The message may influence the recipient's confidence in their ability to improve their performance and achieve their goals.
+Mechanisms are factors that the intervention operates through to influence the feedback recipient. Adapted from [Carey et. al. 2019](https://doi.org/10.1093/abm/kay078). The social approach pathway has the following mechanisms:
```
Comment (mackgalante) [inline vignettes/social_approach_vignette/readme.md:98]: Added link to the Carey et al paper

File: vignettes/social_approach_vignette/readme.md [hunk 2]
```diff
@@ -95,11 +95,11 @@ Moderators are factors that inhibit or promote the influence of the feedback int
...
+1. `Awareness (knowledge)` - The message may change the recipient's awareness of their performance, it's change over time, and of their peer's performance via institutional benchmarks.
```
Comment (mackgalante) [inline vignettes/social_approach_vignette/readme.md:99]: Updated language with information from the paper. Needs approval.

File: vignettes/social_approach_vignette/readme.md [hunk 3]
```diff
@@ -95,11 +95,11 @@ Moderators are factors that inhibit or promote the influence of the feedback int
...
+2. `Subjective norms` - The message may influence the recipient by creating or reinforcing their knowledge of their peer's performance and their relative status within their peer group.
```
Comment (mackgalante) [inline vignettes/social_approach_vignette/readme.md:100]: Updated language, needs approval

File: vignettes/social_approach_vignette/readme.md [hunk 4]
```diff
@@ -95,11 +95,11 @@ Moderators are factors that inhibit or promote the influence of the feedback int
...
+3. `Motivation` - The message may give the recipient purpose or direction towards changing their performance for a given measure.
```
Comment (mackgalante) [inline vignettes/social_approach_vignette/readme.md:101]: Updated language, needs approval

File: vignettes/social_approach_vignette/readme.md [hunk 5]
```diff
@@ -95,11 +95,11 @@ Moderators are factors that inhibit or promote the influence of the feedback int
...
+4. `Optimism` - The message may influence the recipient's confidence that things will happen for the best and they will continue to increase their performance, gaining social status.
```
Comment (mackgalante) [inline vignettes/social_approach_vignette/readme.md:102]: Updated language, needs approval

Comment (mackgalante) [reply vignettes/social_approach_vignette/readme.md:102]: Deleting redundant line reviews, only need this review block to see edits to lines 99-102
Please review the mechanisms above, and let me know if we like the verbiage!

File: vignettes/social_approach_vignette/readme.md (hunk 2 above)
Comment (zachll) [inline vignettes/social_approach_vignette/readme.md:99]: How about:

1. `Awareness (knowledge)` - The message may change the recipient's awareness of their performance level, its change over time, and of their peers' performance via institutional benchmarks.

File: vignettes/social_approach_vignette/readme.md (hunk 3 above)
Comment (zachll) [inline vignettes/social_approach_vignette/readme.md:100]: How about:

2. `Subjective norms` - The message may influence the recipient by creating or reinforcing knowledge of their peers' performance and their relative status within their peer group.

**Review by zachll — COMMENTED** [review]
adding some minor revisions

File: vignettes/social_approach_vignette/readme.md (hunk 2 above)
Comment (mackgalante) [reply vignettes/social_approach_vignette/readme.md:99]: Nice catches, implementing. 
Need to remember grammar errors aren't caught in markdown..

File: vignettes/social_approach_vignette/readme.md (hunk 3 above)
Comment (mackgalante) [reply vignettes/social_approach_vignette/readme.md:100]: Implemented!

**Pull request merged**